src/main/webapp/script.js

```diff
@@ -12,3 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+function onIndexLoad() {
+  checkLogin();
+}
+
+function checkLogin() {
+  fetch('/login').then(response => response.json()).then(status => {
+    if(status.loggedIn){
+      restrictedTabs = document.getElementsByClassName('restricted-tab');
+      for(let element of restrictedTabs) {
```

Comment:
To make this more understandable, just remove "disabled" from the class list of these` restricted-tab`'s.

Reply:
Done